code/Test_definitions/carrier-billing-preparePayment.feature

```diff
@@ -0,0 +1,442 @@
+Feature: CAMARA Carrier Billing API, v0.3 - Operation preparePayment
+  # Input to be provided by the implementation to the tester
+  #
+  # Implementation indications:
+  # * Telco Operator carrier billing behaviour mode: sync or async
+  # 
+  # Testing assets:
+  # * A phone number eligible for payment (no restrictions for it to be used to perform a payment)
+  # * A phone number not-eligible for payment (payment is denied for it due to business conditions)
+  #
+  # References to OAS spec schemas refer to schemas specifies in carrier-billing.yaml, version 0.3.0
+
+  Background: Common preparePayment setup
+    Given the resource "/carrier-billing/v0.3/payments/prepare"                                                              |
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the request body is set by default to a request body compliant with the schema
+
+  ##############################
+  # Happy path scenarios
+  ##############################
+
+
+  @prepare_payment_01_generic_success_scenario
+  Scenario: Common validations for any success scenario
+    # Valid default request body compliant with the schema
+    # Property "$.amountTransaction" is set with required information only
+    Given the request body property "$.amountTransaction" is set with valid required information
+    And the request body is set to a valid request body
+    When the HTTP "POST" request is sent
+    Then the response status code is 201
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    # The response has to comply with the generic response schema which is part of the spec
+    And the response body complies with the OAS schema at "/components/schemas/PaymentCreated"
+
+
+  @prepare_payment_02_generic_success_scenario_with_sink_information
+  Scenario: Common validations for any success scenario with sink information provided
+    # Property "$.amountTransaction" is set with required information only
+    # Property "$.sink" is set with a valid public accessible HTTPs endpoint
+    Given the request body property "$.amountTransaction" is set with valid required information
+    And the request body property "$.sink" is set with a valid URL
+    And the request body is set to a valid request body
+    When the HTTP "POST" request is sent
+    Then the response status code is 201
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    # The response has to comply with the generic response schema which is part of the spec
+    And the response body complies with the OAS schema at "/components/schemas/PaymentCreated"
```

Comment:
Should not be the schema `components/schemas/BodyAmountReservationTransactionForReserveInput`?